package.json

```diff
@@ -14,12 +14,18 @@
         "accept",
         "accept-encoding"
     ],
+    "eslintConfig": {
+        "extends": [
+            "plugin:@hapi/module"
+        ]
+    },
     "dependencies": {
         "@hapi/boom": "9.x.x",
         "@hapi/hoek": "9.x.x"
     },
     "devDependencies": {
         "@hapi/code": "8.x.x",
+        "@hapi/eslint-plugin": "^5.1.0",
```

Comment:
Shouldn't it be kept as `"@hapi/eslint-plugin": "*",` so that it always follows lab's?

Reply:
Good call @Marsup. I agree.